charts/orchestrator-infra/values.yaml

```
@@ -0,0 +1,24 @@
+serverlessLogicOperator:
+  enabled: true  # whether the operator should be deployed by the chart
+  subscription:
+    namespace: openshift-serverless-logic  #  namespace where the operator should be deployed
+    channel: alpha  #  channel of an operator package to subscribe to
+    installPlanApproval: Manual  #  whether the update should be installed automatically
```

```suggestion
    installPlanApproval: Automatic  #  whether the update should be installed automatically
```

Shouldn't this be Automatic by default?

---

Having the installPlan be Automatic will cause it to automatically install updates as well. We want to avoid scenarios where the OpenshiftServerlessLogic operator version gets upgraded, but the newer version isn't supported by other orchestrator functionality.

---

Ok, that makes sense. So there would be an extra step for cluster admins (approving the Install Plan) to not forget to document.